doc/en/_templates/links.html

```diff
@@ -2,7 +2,6 @@ <h3>Useful Links</h3>
 <ul>
   <li><a href="https://pypi.org/project/pytest/">pytest @ PyPI</a></li>
   <li><a href="https://github.com/pytest-dev/pytest/">pytest @ GitHub</a></li>
-  <li><a href="http://plugincompat.herokuapp.com/">3rd party plugins</a></li>
```

Comment:
While it makes sense to remove this from the external links list, I would still like to see a link to this page in the "Table of Contents" side bar:

![image](https://user-images.githubusercontent.com/1085180/96750399-45b43600-13a2-11eb-8ba9-5f4c0fec3887.png)

Reply:
I added it in between `Customize` and `Changelog`. Let me now if that is okay, else pls resolve.